File: Bronze/10808/10808.cpp
// [10808] 알파벳 개수
#include <iostream>
#include <string>
using namespace std;

int main(void)
{
    string str;
    cin >> str;
    
    for (int i = 'a'; i < 'z'+1; i++)
    {
        int count = 0;
        for (int j = 0; j < str.size(); j++)
        {
            if (str[j] == i) { count++; }
        }
        cout << count << " ";
    }
    cout << "\n";
}

File: Bronze/10808/10808.java
// [10808] 알파벳 개수
import java.util.Scanner;

public class Main {
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        String str = sc.next();
        sc.close();

        for (int i = 'a'; i < 'z'+1; i++) {
            int count = 0;
            for (int j = 0; j < str.length(); j++) {
                if (str.charAt(j) == i) count++;
            }
            System.out.print(count + " ");
        }
    }
}